server/events/command_runner.go

```diff
@@ -416,6 +416,10 @@ func (c *DefaultCommandRunner) updatePull(ctx *CommandContext, command PullComma
 	if err := c.VCSClient.CreateComment(ctx.BaseRepo, ctx.Pull.Num, comment); err != nil {
 		ctx.Log.Err("unable to comment: %s", err)
 	}
+
+	if err := c.VCSClient.HideOldComments(ctx.BaseRepo, ctx.Pull.Num); err != nil {
```

Comment:
The method should be `HidePreviousPlanComments`

Reply:
Shouldn't this run **before** the CreateComment call? Otherwise it will hide the latest comment.

Reply:
Oops, correct.